db/migrate/20211117094503_change_collation_for_tag_names.acts_as_taggable_on_engine.rb

```diff
@@ -0,0 +1,6 @@
+# This migration comes from acts_as_taggable_on_engine (originally 5)
+# This migration is added to circumvent issue #623 and have special characters
+# work properly
+class ChangeCollationForTagNames < ActiveRecord::Migration[4.2]
```

Comment:
Should we add this file in our project then? It does nothing and in few years later I will be a bit confused

Reply:
The point is to avoid migration to be auto-generated by the gem going forward.

Reply:
How could it be autogenerated? You have to run a rails command no? And also we review so we should just discard it. It does not make sense to commit empty migration.

Reply:
When we upgrade the gem and run generation of migrations, we will always have to look at it and discard it as unnecessary. So my intention is to just avoid having to look at it going forward. Is there any other way to achieve this?

Reply:
I mean, when one does:
```
rake acts_as_taggable_on_engine:install:migrations
```

Then missing migrations are generated. And when this class is missing, then it will be generated. So one will have to investigate whether it is a new one or old one and whether it applies. Maybe not a huge task but I wanted to avoid this. WDYT?

Reply:
I think it is unnecessary, you do not want to commit unnecessary code
We will anyway investigate if we need to run this command again in the future

Reply:
We don't need this migration because it is a noop for our database state. So we will never need it.